tests/Hedgehog.Linq.Tests/LinqTests.cs

```diff
@@ -94,9 +94,9 @@ public void CanUseWhereWithAssertion()
         public void CanUseWhereWithBool()
         {
             var property =
-                from x in ForAll(Gen.Bool)
+                from x in ForAll(Gen.FromValue(true))
                 where x == true
-                from y in ForAll(Gen.Bool)
+                from y in ForAll(Gen.FromValue(false))
                 where y == false
                 select x && !y;
 
```

Comment:
Perhaps we can take this a step further and just remove the call to property `check` so that those don't actually run (that is, we only care that the syntax works). It doesn't have to be done now, it's just a thought.